components/sensor/ld2420.rst

```diff
@@ -0,0 +1,261 @@
+      g15_still_threshold: 100
+
+.. note::
+
+    For UART configuration, ``baud_rate``, ``parity`` and ``stop_bits`` **must be** respectively ``115200 or 256000``, ``NONE`` and ``1``.
+    Use of hardware UART pins is critical in order to support the out-of-the-box 256000 or 115200 baud rates of the LD2420 sensor.
+
+Configuration variables:
+************************
+
+The configuration is made up of three parts: The LD2420 component, individual sensors,
+and binary sensors. These may change, this is a early component release based on firmware v1.5.3.
+*There are clearly undocumented functions in the firmware which are not available at this time.*
+
+- **presence_time_window** (*Optional*, int): Time in seconds during which presence state will stay present
+  after leaving. Any move or still energy detection within the time window restarts a count down from the set value.
+  Defaults to ``120s``.
+- **detection_gate_min** (*Optional*, int): Minimum distance for move or still energy detection.
+  Value between gate ``1`` and ``detection_gate_max - 1`` incrementally (each increment equals ``0.75m`` at this time).
+  Defaults to ``1``.
+- **detection_gate_max** (*Optional*, int): Maximum gate for movement detection.
+  Value between ``1`` to ``15`` each gate detects movement and still energy within ``0.75m`` of range incrementally.
+  Defaults to ``12``.
+  With maximum distance equates to ``16`` * ``0.75m`` resulting in a possible ``12m`` however accuracy at ``12m`` is not certain.
+  The recommended maximum value is ``12`` * ``0.75`` which results in approximately ``9m``.
+- **g#_move_threshold** (*Optional*, int): Gate threashold level for motion energy detection (Gate # from 0 to 15).
+  Above this level for the considered gate (distance), movement detection will be triggered.
+  Defaults to ``see table below``.
+- **g#_still_threshold** (*Optional*, int): Threshold for the Xth gate for still detection. (Gate # from 0 to 15).
+  Above this level for the considered gate (distance), still detection will be triggered.
+  Defaults to ``see table below``.
+
+.. list-table:: Default values for gate threshold
+    :widths: 25 25 25
+    :header-rows: 1
+
+    * - Gate:
+      - Default: Move threshold
+      - Default: Still threshold
+    * - 0
+      - 60000
+      - 40000
+    * - 1
+      - 30000
+      - 20000
+    * - 2
+      - 400
+      - 200
+    * - 3
+      - 300
+      - 250
+    * - 4
+      - 250
+      - 150
+    * - 5
+      - 250
+      - 150
+    * - 6
+      - 250
+      - 150
+    * - 7
+      - 250
+      - 150
+    * - 8
+      - 300
+      - 150
+    * - 9
+      - 250
+      - 150
+    * - 10
+      - 250
+      - 150
+    * - 11
+      - 250
+      - 150
+    * - 12
+      - 250
+      - 100
+    * - 13
+      - 200
+      - 100
+    * - 14
+      - 200
+      - 100
+    * - 15
+      - 200
+      - 100
+
+Sensor
+------
+
+The ``ld2420`` sensor allows you to use your :doc:`ld2420` to sense mmWave distance measurements.
```

Comment:
I think this might've gotten paved when you reworked the docs post-refactoring. This is just a suggestion; I just think the way it is phrased here is quite awkward.